### PR TITLE
Update ar-AR.json

### PR DESCRIPTION
**Pull request by hg13bs**
more strings lol

### CODE REVIEW

**Review by Ushie — CHANGES_REQUESTED** [review]
What is the English String for this? the translation seems incorrect to me, what is "Reaction Dev Tools"?

File: ar-AR.json [hunk 1]
```diff
@@ -5,15 +5,23 @@
...
   "ENMITY_DEVELOPMENT_SETTINGS": "إعدادات البرمجة",
+  "ENMITY_ENABLED": "مفعل",
   "ENMITY_ERROR_BOUNDARY": "حدود الخطأ",
   "ENMITY_ERROR_BOUNDARY_DESC": "وحدة استعادة الأعطال. لا تقم بتعطيله إذا كنت مستهلكًا.",
   "ENMITY_GENERAL": "عام",
   "ENMITY_INSTALL": "تثبيت",
   "ENMITY_INSTALL_TITLE": "تثبيت {type}",
+  "ENMITY_LOADER": "المحمل",
+  "ENMITY_LOADER_CUSTOM_BUNDLE": "عنوان الurl للحزمة المخصصة",
+  "ENMITY_LOADER_DEVTOOLS": "رد فعل DevTools",
```
Comment (Ushie) [inline ar-AR.json:20]: What is the English String for this? the translation seems incorrect to me, what is "Reaction Dev Tools"?

**Review by Ushie — CHANGES_REQUESTED** [review]

File: ar-AR.json (hunk 1 above)
Comment (hg13bs) [reply ar-AR.json:20]: err the english string is React DevTools

Comment (Ushie) [reply ar-AR.json:20]: React is a name, it should either remain in English or be translated into the pronunciation of it in Arabic

**Review by Ushie — CHANGES_REQUESTED** [review]
If possible, do you mind if you translate DevTools? there is a possible translation so I don't think there's a reason to keep them in English

**Comment by hg13bs** [comment]
hmm ok

**Comment by hg13bs** [comment]
is this change fine?

**Pull request merged**